src/control/server/instance.go

```diff
@@ -0,0 +1,324 @@
+//
```

Comment:
Yes. The ioserver package is just for configuring and managing an instance of daos_io_server without any of the control plane baggage. The reason we keep the stuff in instance.go in the server package is because this is where all of the control-plane-specific stuff lives (superblock, RPC, etc). I need to flesh things out more with the privileged binary work, but the goal is to be able to test all of the harness/instance stuff with a mock runner.